src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/version/fixtures.js

```diff
@@ -0,0 +1,81 @@
+export const CPSVersionBlock = {
+  id: '63501296',
+  subType: 'primary',
+  format: 'video',
+  externalId: 'journalism_world_service_stream_01',
+  duration: 'PT0S',
+  caption: 'live stream from event',
+  image: {
+    id: '63501296',
+    subType: 'thumbnail',
+    href: 'http://b.files.bbci.co.uk/15E0C/test/_63721698_p01kx435.jpg',
+    path: '/cpsdevpb/15E0C/test/_63721698_p01kx435.jpg',
+    height: 360,
+    width: 640,
+    altText: 'connectionAltText',
+    copyrightHolder: 'BBC',
+    type: 'image',
+  },
+  embedding: false,
+  available: true,
+  live: true,
+  type: 'version',
+};
+
+export const optimoVersionBlock = {
+  type: 'version',
+  model: {
+    locator: 'urn:bbc:pips:pid:63501296',
+    blocks: [
+      {
+        type: 'aresMedia',
+        model: {
+          blocks: [
+            {
+              type: 'aresMediaMetadata',
+              blockId: 'urn:bbc:ares::primary:63501296',
+              model: {
+                embedding: false,
+                format: 'audio_video',
```

Comment:
Giving the CPSVersionBlock's ` format` is `video` should this be `video` rather than `audio_video`?

Reply:
"audio_video" is the correct format for Optimo blocks - https://github.com/bbc/simorgh/search?p=1&q=audio_video&unscoped_q=audio_video

I remember having to add this to the media transformer too - it doesn't work as "video" :(

Reply:
Ahhh okay, I was thinking that the CPS block would be `format: audio` and we'd make it the format `audio_video` in the Optimo blocks